assets/js/components/NewMetricBadge.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * NewMetricBadge component.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Tooltip } from 'googlesitekit-components';
+import Badge from './Badge';
+import Link from './Link';
+
+function NewMetricBadge( { tooltipTitle, learnMoreLink } ) {
```

Comment:
I realize this is how it was outlined in the IB but is there any reason we can't have a more generic "New" badge? Nothing here seems to be specific to a new metric that we wouldn't want for another use in a different context. If not, I'd suggest we make this slightly more generic.